.appveyor.yml

```diff
@@ -6,7 +6,7 @@ environment:
 
   BINSTAR_TOKEN:
     # The BINSTAR_TOKEN secure variable. This is defined canonically in conda-forge.yml.
-    secure: MP4hZYylDyUWEsrt3u3cod2sbFeRwUziH02mvQOdbjsTO/l1yIxDkP/76rSIjcGC
+    secure: tumuXLL8PU75WMnRDemRy02ruEq2RpNxeK3dz0MjFssnosPm2v4EFjfNB4PTotA1
```

Comment:
This is what we missed to get the packages updated. @isuruf did an amazing job of updating all the tokens for Travis-CI and AppVeyor on the `master` branch but we could no propagate that to the branches, so we need to do a few of those manually by just copying the `conda-forge.yaml` from `master` and re-rendering.